apps/fishing-map/features/datasets/upload/datasets-upload.hooks.tsx

```diff
@@ -129,11 +129,20 @@ export function useDatasetMetadataOptions(
             schema?.type === 'boolean' ||
             (schema?.type === 'string' && schema?.enum && schema?.enum?.length > 0)
           const isRangeAllowed = schema?.type === 'range' && schema.enum?.length === 2
+          const isMaxValuesExceeded = schema.enum?.[0] === 'maximum values exceeded'
```

Comment:
Why not move this to a const so we can also reuse it [here](https://github.com/GlobalFishingWatch/frontend/pull/3446/files#diff-a031c0dd30bf490db7179a04cec0d01cd2ca882b3c734f2488eab3fa897d5cd4R100), which helps to understand the relationship